static/docs/command-reference/get.md

```diff
@@ -66,12 +67,12 @@ created in the current working directory, with its original file name.
 
 - `-v`, `--verbose` - displays detailed tracing information.
 
-## Examples
+## Example: Retrieve a model from a DVC remote
```

Comment:
> from a DVC remote

To the user this may not be obvious. They don't indicate the remote but the URL (or path) to the DVC repo. So maybe say "from an external DVC repository" or just leave "Examples". Open to suggestions.

Reply:
How about:

```suggestion
## Example: Download a file tacked by an external DVC repository
```